### PR TITLE
Update primitive restart readme with invalid models notification

### PR DESCRIPTION
**Pull request by stevk**

### CODE REVIEW

File: Output/Mesh_PrimitiveRestart/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-These models are intended to test behavior of primitive restart feature of graphics APIs.
+These models are intended to test behavior of primitive restart feature of graphics APIs. These are not valid glTF models as primitive restart values are disallowed. [See the glTF specification](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#primitiveindices) for more details.
```
Comment (bghgary) [inline Output/Mesh_PrimitiveRestart/README.md]: @lexaknyazev I noticed that the spec property reference has text that doesn't exist in the schema (the Minimum and Maximum). Did you add these by hand?

Comment (bghgary) [reply Output/Mesh_PrimitiveRestart/README.md]: Ah, I looked at that and for some reason still thought it was you. 😉

cc @donmccurdy

Comment (donmccurdy) [reply Output/Mesh_PrimitiveRestart/README.md]: @lexaknyazev what do you mean by rearranged? Sounds like we need to add the min/max in the schema?

**Comment by donmccurdy** [comment]
I would consider making it more prominent that these are invalid glTF files.

File: Output/Mesh_PrimitiveRestart/README.md (hunk 1 above)
Comment (lexaknyazev) [reply Output/Mesh_PrimitiveRestart/README.md]: Schema-defined fields refer to `primitive.indices` _property_, not accessor's content.

**Comment by bghgary** [comment]
> I would consider making it more prominent that these are invalid glTF files.

Do you have a recommendation? Currently, the `manifest.json` has a flag that indicates invalid glTF assets. This PR adds wording that indicate this for this group.

File: Output/Mesh_PrimitiveRestart/README.md (hunk 1 above)
Comment (donmccurdy) [reply Output/Mesh_PrimitiveRestart/README.md]: Ahhh ok yes, my mistake. 😅

**Comment by donmccurdy** [comment]
Bold or block quote text would be enough I think. If they appear in the viewer some UI would be helpful there, too.

I could maybe see an argument for not keeping the invalid models in the repo, but having a flag to generate them as needed. Unclear what most users would do with these.

File: Output/Mesh_PrimitiveRestart/README.md (hunk 1 above)
Comment (bghgary) [reply Output/Mesh_PrimitiveRestart/README.md]: Looks like we all overlooked where the text was being added when reviewing the PR. 😕

**Comment by bghgary** [comment]
> Bold or block quote text would be enough I think.

👍 

> If they appear in the viewer some UI would be helpful there, too.

Perhaps the View links should be removed for these assets.

> I could maybe see an argument for not keeping the invalid models in the repo, but having a flag to generate them as needed. Unclear what most users would do with these.

Yeah, we've discussed this. We considered putting these in a completely separate folder.

File: Output/Mesh_PrimitiveRestart/README.md (hunk 1 above)
Comment (donmccurdy) [reply Output/Mesh_PrimitiveRestart/README.md]: Opened https://github.com/KhronosGroup/glTF/pull/1532

**Pull request merged**